user_docs/en/userGuide.t2t

```diff
@@ -4328,6 +4328,10 @@ Secure mode disables:
 Installed copies of NVDA store their configuration including add-ons in ``%APPDATA%\nvda``.
 To prevent NVDA users from modifying their configuration or add-ons directly, user access to this folder must also be restricted.
 
+Secure mode is ineffective for portable copies of NVDA.

```

Comment:
```suggestion
Secure mode is ineffective for portable copies of NVDA.
This limitation also apply to the temporary portable copy which is running when launching the installer; this temporary portable copy is executed from a subfolder of the user's temporary folder.
```

Reply:
```suggestion
Secure mode is ineffective for portable copies of NVDA.
This limitation also applies to the temporary copy of NVDA which runs when launching the installer.
```

The second part is not quite true. If NVDA is already installed, the launcher config directory is the installed config directory. I think we should keep this statement more broad, as the issue only really exists when running the launcher without a previous installation.